phys/module_cu_ntiedtke.F

```diff
@@ -590,16 +584,20 @@ subroutine tiecnvn(pu,pv,pt,pqv,pqc,pqi,pqvf,ptf,poz,pzz,pomg, &
       integer i,j,k,lq,km,km1
       real dt,ztpp1
       real zew,zqs,zcor
-      real scale_fac(lq), dxref
+      real scale_tau(lq), scale_mf(lq), relax_t_limit(lq), dxref
 !
 !  set scale-dependency factor when dx is < 15 km
 !
       dxref = 15000.
       do j=1,lq
-      if (dx(j).lt.dxref .and. ntiedtke_dx_opt.eq.1) then
-          scale_fac(j) = (1.+log(dxref/dx(j)))**3
+      if (dx(j).lt.dxref) then
+          scale_tau(j) = (1.+log(dxref/dx(j)))**3
```

Comment:
A few issues here:
1) There is inconsistency at 15 km grid spacing. scale_fac = 1. and 1.1995, respectively. 
2) There is inconsistency for relaxing time limit as well, 43200. vs. 10800.
I prefer the name scale_fac. Maybe better to use the name scale_fac2 for shallow convection.